prow/images/buildpack-golang/Dockerfile

```diff
@@ -43,8 +43,9 @@ RUN curl -fLSs -o go.tar.gz "https://dl.google.com/go/go${GO_VERSION}.linux-${AR
     curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 
 # Install kubebuilder
```

Comment:
Please move this comment closet to the corresponding ENV line

Reply:
Fixed